ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh

```diff
@@ -240,3 +240,9 @@ oc exec -it  pod/mariadb-openstack -- mysql -uroot -p${MARIADB_PASSWD} -e "show
 # Post tests for keystone-operator
 # Check to confirm you can issue a token.
 openstack token issue
+
+# NOTE: if manila is deployed, build a default share required by tempest
+if [[ "${SERVICE_NAME}" == "MANILA" ]]; then
+    oc exec -it pod/openstackclient -- openstack share type create default false
```

Comment:
@cescgina maybe if we run the command by using the `openstackclient` pod it works because it's able to discover the right endpoints and should be aware about `share` service.